lib/omniauth/strategies/drip.rb

```diff
@@ -38,6 +38,10 @@ def user_info
       def raw_info
         @raw_info ||= JSON.parse(access_token.get("/v2/accounts").body)
       end
+
+      def callback_url
+        options[:redirect_uri] || (full_host + script_name + callback_path)
+      end
```

Comment:
I don't know about the other change, but without this change this gem is essentially broken on omniauth-oauth2 v1.4+